packages/mdc-auto-init/index.js

```diff
@@ -19,6 +19,27 @@ const registry = Object.create(null);
 const CONSOLE_WARN = console.warn.bind(console);
 
 /**
+ * Fires a cross-browser-compatible custom event with the given data
+ * @param {string} evtType
+ * @param {!Object} evtData
+ * @param {boolean=} shouldBubble
+ */
+export function emit(evtType, evtData, shouldBubble = false) {
```

Comment:
Could we consider making this a **private** method?
My concern is there might not be a use case where this function will be called by external user (now). 

We could do `function emit_` to identify this is a private method and remove `export`. I also commented about revoke and test below.

What do you think?